webapp/layouts/error.vue

```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="error-container">
+    <img class="error-image" :src="image" />
+    <br />
+    <span class="error-message">{{ $t(message) }}</span>
```

Comment:
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2020-01-24T13:51:28Z" title="Friday, January 24th 2020, 2:51:28 pm +01:00">Jan 24, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="error-container">
+    <base-icon class="error-icon" name="exclamation-circle" />
+    <br />
+    <span v-if="error.message === 'This page could not be found'" class="error-message">
```


I was not able to locate the place where the default 404 error message comes from. So I catch it here. Any ideas?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-24T14:13:23Z" title="Friday, January 24th 2020, 3:13:23 pm +01:00">Jan 24, 2020</time>_

---


You could dispatch the error code:

```js
computed: {
  message() {
    const mapping = {
       403: 'error-pages.not-authorized',
       404: // ...
       500: // ...
    }
    const key = mapping(this.error.code) || 'error.default'
    return this.$(key)
  }
}